src/lib/WireGuard.js

```diff
@@ -60,18 +59,18 @@ module.exports = class WireGuard {
         }
 
         await this.__saveConfig(config);
-        await Util.exec(`wg-quick down ${WG_INTERFACE}`).catch(() => { });
-        await Util.exec(`wg-quick up ${WG_INTERFACE}`).catch((err) => {
-          if (err && err.message && err.message.includes(`Cannot find device ${WG_INTERFACE}`)) {
-            throw new Error(`WireGuard exited with the error: Cannot find device ${WG_INTERFACE}\nThis usually means that your host's kernel does not support WireGuard!`);
+        await Util.exec('wg-quick down wg0').catch(() => { });
+        await Util.exec('wg-quick up wg0').catch((err) => {
+          if (err && err.message && err.message.includes('Cannot find device "wg0"')) {
+            throw new Error('WireGuard exited with the error: Cannot find device "wg0"\nThis usually means that your host\'s kernel does not support WireGuard!');
           }
 
           throw err;
         });
-        // await Util.exec(`iptables -t nat -A POSTROUTING -s ${WG_DEFAULT_ADDRESS.replace('x', '0')}/24 -o ' + WG_DEVICE + ' -j MASQUERADE`);
+        // await Util.exec(`iptables -t nat -A POSTROUTING -s ${WG_DEFAULT_ADDRESS.replace('x', '0')}/24 -o eth0 -j MASQUERADE`);
```

Comment:
Unsubstantiated observation, but I wonder whether or not this was omitted from the original PR where the ability for changing interface was introduced. Or is this similar to the hardcoded ports, i.e. must always be set to `eth0`?

Reply:
Done: https://github.com/wg-easy/wg-easy/pull/714/commits/d16ed82290050fbf383f2a63b5a5ee8fe6337cf6